modules/twinkleconfig.js

```diff
@@ -1399,88 +1396,79 @@ Twinkle.config.listDialog.display = function twinkleconfigListDialogDisplay(e) {
 	dialog.setTitle(curpref.label);
 	dialog.setScriptName('Twinkle preferences');
 
-	var dialogcontent = document.createElement('div');
-	var dlgtable = document.createElement('table');
-	dlgtable.className = 'wikitable';
-	dlgtable.style.margin = '1.4em 1em';
-	dlgtable.style.width = 'auto';
-
-	var dlgtbody = document.createElement('tbody');
-
-	// header row
-	var dlgtr = document.createElement('tr');
-	// top-left cell
-	var dlgth = document.createElement('th');
-	dlgth.style.width = '5%';
-	dlgtr.appendChild(dlgth);
-	// value column header
-	dlgth = document.createElement('th');
-	dlgth.style.width = '35%';
-	dlgth.textContent = curpref.customListValueTitle ? curpref.customListValueTitle : 'Value';
-	dlgtr.appendChild(dlgth);
-	// label column header
-	dlgth = document.createElement('th');
-	dlgth.style.width = '60%';
-	dlgth.textContent = curpref.customListLabelTitle ? curpref.customListLabelTitle : 'Label';
-	dlgtr.appendChild(dlgth);
-	dlgtbody.appendChild(dlgtr);
+	var $dlgtbody;
+
+	dialog.setContent(
```

Comment:
Dunno if you tested this, but actually running it will fail here: `Uncaught TypeError: Node.appendChild: Argument 1 does not implement interface Node.`

Pretty sure this needs to be `$('<div>').append(...snip...)[0]` to properly `setContent`

Reply:
Doing this worked in my testing, so I can just do this if you'd like.

Reply:
I though I tested this, but now that I think of it, patch-test.js wouldn’t really work for config.

Reply:
Works fine for me?